addons/event/views/event_slot_views.xml

```diff
@@ -0,0 +1,80 @@
+<?xml version="1.0"?>
+<odoo><data>
+
+    <record id="view_event_slot_form" model="ir.ui.view">
+        <field name="name">event.slot.form</field>
+        <field name="model">event.slot</field>
+        <field name="arch" type="xml">
+            <form>
+                <group>
+                    <group>
+                        <field name="start_hour" string="From" widget="float_time"/>
+                        <field name="color" widget="color_picker"/>
```

Comment:
Actually, this is just meant to be displayed in the calendar? Or does it have impact on the ticket templates or whatever?